docs/GetStarted/getting_started_android_cmake.md

```diff
@@ -82,7 +82,7 @@ $ cmake -G Ninja -B build-android  \
     for your target device. You can also refer to Android NDK's
     [CMake documentation](https://developer.android.com/ndk/guides/cmake) for
     more toolchain arguments.
-*   Building IREE compilers, tests, and samples for Android is not supported at
+*   Building IREE compilers and samples for Android is not supported at
     the moment; they will be enabled soon.
```

Comment:
Do we actually plan on getting the compiler (and LLVM) building for Android? Are we going to use that?

Reply:
Maybe. :) Not immediately for IREE; but I can envision that projects might want to reuse IREE compilers to do online conversion. :)